exercises/fulfill_a_promise/problem.md

```diff
@@ -1,50 +1,41 @@
 # What is a promise?
-
-A promise is an object that defines a method called `then`.
-The promise object represents a value (or values) that may
-be available some time in the future.
-
-When a promise is resolved, any *"success functions"* registered
-with the `then` method will be called with the newly available
-data as arguments.
-
-If a promise is rejected then any *"failure functions"* registered
-with the `then` method will be called with the `Error` as argument.
-
-For now, do not be concerned with exactly **how** this works or what
-the syntax is.  We are about to dive into that in detail.
+A promise is an object that lets a consumer attach success and failure
+handlers to an asyncrhonous operation. If the operation succeeds,
+the function you register using the promises `then` function
+will be called with the result of the operation. If the operation
+fails, the function you register with the promises
+`catch` function is called.
+
+Just as the promise consumer can register for success or failure
+of the asynchronous operation, the producer of a promise needs to
+communicate success or failure. When the operation completes successfully,
+the producer calls a `resolve` function with the result of the operation.
+If the operation failed, the producer is responsible for calling the
+`reject` function with the error describing the failure.
 
 ## Setup
 
-To do many of the lessons in this workshop, you will need to install
-the `q` module.  To do this, type the following in your project's
-directory:
-
+The `q` module is the promise library used for this set of exercises.
+Install it in your project directory using:
 ```sh
-$ npm install q --save
+$ npm install q
 ```
 
-This will install a local copy of the `q` module into your directory
-which you can import into the file you have written using:
+You can now reference it in your JavaScript file using:
 
 ```js
-var q = require('q');
+var q = require("q");
```

Comment:
eh? hint wants single. Anyway probably best to leave as-is for already written code :smile_cat: